packages/components/src/internal/components/entities/EntityTypeDeleteConfirmModal.tsx

```diff
@@ -45,6 +46,11 @@ export class EntityTypeDeleteConfirmModal extends React.Component<Props, any> {
             >
                 <span>
                     The {noun.toLowerCase()} type and all of its {dependencies} will be permanently deleted.
+                    {isShared &&
```

Comment:
I haven't pulled down this fb to look at it, but do you think this warrants putting this text in something like an info/warning alert banner to make it stand out?

Reply:
I consulted with Andrew on this, and the inclusion of the update in the title and the extra sentence here seemed sufficient. An alert in this small of a modal seems a bit "heavy" to me.

Reply:
Sounds good. Thanks.